build-tools/packages/build-cli/src/commands/check/latestVersions.ts

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Args } from "@oclif/core";
+import * as semver from "semver";
+import { findPackageOrReleaseGroup, packageOrReleaseGroupArg } from "../../args.js";
+import { BaseCommand, sortVersions } from "../../library/index.js";
+
+export default class LatestVersionsCommand extends BaseCommand<typeof LatestVersionsCommand> {
+	static readonly summary =
+		"Determines if an input version matches a latest minor release version.";
+
+	static readonly description =
+		"This command is used in CI to determine if a pipeline was triggered by a release branch with the latest minor version of a major version.";
+
+	static readonly args = {
+		version: Args.string({
+			required: true,
+			description: "The version corresponding to the pipeline trigger branch.",
+		}),
+		package_or_release_group: packageOrReleaseGroupArg(),
+	} as const;
+
+	public async run(): Promise<boolean> {
+		const { args } = this;
+		const context = await this.getContext();
+		const versionInput = this.args.version;
+
+		if (args.package_or_release_group === undefined) {
+			this.error("No dependency provided.");
+		}
```

Comment:
Shouldn't be needed; since the arg is required, oclif should make the type of `args.package_or_release_group` a string, not `string | undefined`